spec/src/modules/search.js

```diff
@@ -656,4 +656,477 @@ describe(`ConstructorIO - Search${bundledDescriptionSuffix}`, () => {
+        apiKey: voiceSearchApiKey,
+        fetch: fetchSpy,
+      });
+
+      search.getVoiceSearchResults(voiceSearchQuery, {
+        section,
+        resultsPerPage,
+      }).then((res) => {
+        const requestedUrlParams = helpers.extractUrlParamsFromFetch(fetchSpy);
+
+        expect(res).to.have.property('request').to.be.an('object');
+        expect(res).to.have.property('response').to.be.an('object');
+        expect(res).to.have.property('result_id').to.be.an('string');
+        expect(res.request.num_results_per_page).to.equal(resultsPerPage);
+        expect(res.response).to.have.property('results').to.be.an('array');
+        expect(res.response.results.length).to.equal(resultsPerPage);
+        expect(requestedUrlParams).to.have.property('num_results_per_page').to.equal(resultsPerPage.toString());
+        done();
+      });
+    });
+
+    it('Should return a response with a valid query, section, and fmtOptions', (done) => {
+      const fmtOptions = { groups_max_depth: 2, groups_start: 'current' };
+      const { search } = new ConstructorIO({
+        apiKey: voiceSearchApiKey,
+        fetch: fetchSpy,
+      });
+
+      search.getVoiceSearchResults(voiceSearchQuery, {
+        section,
+        fmtOptions,
+      }).then((res) => {
+        const requestedUrlParams = helpers.extractUrlParamsFromFetch(fetchSpy);
+
+        expect(res).to.have.property('request').to.be.an('object');
+        expect(res).to.have.property('response').to.be.an('object');
+        expect(res).to.have.property('result_id').to.be.an('string');
+        expect(res.request.fmt_options).to.have.property('groups_max_depth').to.equal(fmtOptions.groups_max_depth);
+        expect(res.request.fmt_options).to.have.property('groups_start').to.equal(fmtOptions.groups_start);
+        expect(requestedUrlParams).to.have.property('fmt_options');
+        expect(requestedUrlParams.fmt_options).to.have.property('groups_max_depth').to.equal(Object.values(fmtOptions)[0].toString());
+        expect(requestedUrlParams.fmt_options).to.have.property('groups_start').to.equal(Object.values(fmtOptions)[1]);
+        done();
+      });
+    });
+
+    it('Should return a response with a valid query, section, and sortBy', (done) => {
+      const sortBy = 'relevance';
+      const { search } = new ConstructorIO({
+        apiKey: voiceSearchApiKey,
+        fetch: fetchSpy,
+      });
+
+      search.getVoiceSearchResults(voiceSearchQuery, {
+        section,
+        sortBy,
+      }).then((res) => {
+        const requestedUrlParams = helpers.extractUrlParamsFromFetch(fetchSpy);
+
+        expect(res).to.have.property('request').to.be.an('object');
+        expect(res).to.have.property('response').to.be.an('object');
+        expect(res).to.have.property('result_id').to.be.an('string');
+        expect(res.request.sort_by).to.equal(sortBy);
+        expect(requestedUrlParams).to.have.property('sort_by').to.equal(sortBy);
+        done();
+      });
+    });
```

Comment:
It might be good to know if/when passing in sort options become [disallowed](https://docs.constructor.io/rest_api/search/natural_language_search/#http-request) but since the behaviour is not intended to be supported to begin with, should we remove this test?